docs/api-machinery/capability-discovery.md

```diff
@@ -33,7 +33,7 @@ Use the constructor(s) from `discovery` package to get a query client.
 ```go
 import (
     "sigs.k8s.io/controller-runtime/pkg/client/config"
-    "github.com/vmware-tanzu/tanzu-framework/pkg/v1/sdk/capabilities/discovery"
+    "github.com/vmware-tanzu/tanzu-framework/capabilities/discovery"
```

Comment:
we should separate our our project (TF) imports from outside package (sigs.k8s.io) imports with an empty line.

Reply:
`make fmt` should automatically do this for you.

Reply:
this is in the documentation, didn't make many changes except for updating the package path. Anyway, I did push a change to address this..

Reply:
ok, looks like the mdlint doesn't like the empty line, switching it back to the original state.. don't want to complicate things here by suppressing linter..